Tools/autotest/quadplane.py

```diff
@@ -2164,6 +2164,90 @@ def WindEstimateConsistency(self):
                 self.progress("Wind estimates correlated")
                 break
 
+    def FullRateClimbout(self):
+        '''test the full rate climbout after VTOL Takeoff functions'''
+        # Set from examining the autotest with enable/disable flag.
+        ALTITUDE_CUTOFF_M = 35
+        CRUISE_ALTITUDE_CUTOFF = 120
+        WAIT_AFTER_CRUISE_DELAY_S = 10
```

Comment:
We usually have constants like this declared and defined just before their first (and typically only...) use.  Saves having to go look stuff up all the time.